index.html

```diff
@@ -17,10 +17,44 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="header-background"></div>
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1 class="header">Code Journal</h1>
+          </div>
+        </div>
+      </div>
```

Comment:
I would do something like this instead. Now `header-background` will automatically take up 100% width of the browser (block level), and you don't need to mess with positioning

```
<div class="header-background">
   <div class="container">
      <div class="row">
         <div class="column-full">
            <h1 class="header">Code Journal</h1>
         </div>
      </div>
   </div>
</div>
```